src/backend/main/py.main/caliopen_main/message/core/raw.py

```diff
@@ -54,3 +62,7 @@ class UserRawLookup(BaseUserCore):
 
     _model_class = ModelUserRawLookup
     _pkey_name = 'raw_msg_id'
+
+    @classmethod
+    def message_belongs_to_user(cls, user_id, raw_msg_id):
+        return super(UserRawLookup, cls).object_belongs_to_user(user_id, raw_msg_id)
```

Comment:
no super needed, inheritance permit to have method in class definition directly.

AKA cls.object_belong_to_user exists